cmd/experimental/kueue-populator/charts/kueue-populator/README.md

```diff
@@ -70,6 +88,39 @@ kueuePopulator:
 helm install kueue-populator ./charts/kueue-populator --namespace kueue-system --create-namespace --wait -f my-values.yaml
 ```
 
+### Installing with Topology and ResourceFlavor
+
+To enable Topology Aware Scheduling and configure the default ResourceFlavor with node labels, you can use `--set` flags.
+
+> When using `--set` for keys containing dots (e.g., `cloud.google.com/gke-nodepool`), you must escape the dots with a backslash.
+
+```bash
+helm install kueue-populator ./charts/kueue-populator \
+  --namespace kueue-system \
+  --create-namespace \
+  --wait \
+  --set kueuePopulator.config.topology.levels[0].nodeLabel="cloud\.google\.com/gke-nodepool" \
+  --set kueuePopulator.config.resourceFlavor.nodeLabels."cloud\.google\.com/gke-nodepool"=default-pool
+```
+
+For complex configurations, it is recommended to use a values file:
```

Comment:
nit, This seems already cumbersome to use, so I would recommend installation with the `populator-config.yaml` already. Then we can say, "For simple configuration you may also use minimalistic command..." (basically revert the order)